data_processing/pathology/cli/visualize_tile_labels.py

```diff
@@ -69,7 +71,23 @@ def visualize_tile_labels_with_container(cohort_id: str, container_id: str, meth
         output_dir = os.path.join(os.environ['MIND_GPFS_DIR'], "data", container._namespace_id, container._name, method_id)
         if not os.path.exists(output_dir): os.makedirs(output_dir)
 
-        properties = visualize_scoring(image_node.data, label_node.data, output_dir, method_data)
+        # properties = visualize_scoring(image_node.data, label_node.data, output_dir, method_data)
+        if method_data.get("dsa_config", None):
+            params = label_node.properties
+
+            params["column"]            = "tumor_score"
+            params["input"]             = label_node.properties["data"]
+            params["output_folder"]     = output_dir
+            params["output"]   = method_id
```

Comment:
New changes that are coming: removed `output_folder`, and renamed `output` to `annotation_name`. https://github.com/msk-mind/data-processing/blob/f56a8cec0c755d559eb0cb77ce15a55d3227ad61/data_processing/pathology/cli/dsa/examples/example_heatmap_config.json

Reply:
I've updated the params to match the updates, thanks!